src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,356 @@
+initiate a payment from a payer using only the payer's PayID. 
+
+--- middle
+
+# Terminology
+
+This protocol can be referred to as the `PayId Easy Checkout Protocol`. It uses the following terminology:
+   
+* PayID client: a client that queries a PayID server using the PayID Protocol as defined in [PAYID-PROTOCOL][].
+* PayID server: the endpoint that returns payment account(s) information, which conforms to the PayID Protocol.
+* Merchant: Individual or entity receiving a payment (e.g., e-commerce merchant, charity).
+* Payer: Individual or entity originating a payment to a `merchant`.
+* Wallet: A device or application that holds funds (may be a non-custodial wallet).
+* PayID Easy Checkout URL: The URL that is the result of the PayID Easy Checkout protocol; can be used to redirect a client to a wallet corresponding to a particular PayID.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol that allows an online merchant to
+request a payment from a payer using only the payer's PayID. Implementations
+of the protocol should require little to no server-side engineering efforts, while creating an improved and uniform
+user experience for payers.
+
+The main focus of the Protocol is on PayID Easy Checkout Discovery, which defines how a PayID client can use a PayID
+to retrieve a PayID Easy Checkout URL which represents a resource that the payer's digital wallet can use to initiate 
+a payment to the merchant. 
+
+Though the [appendix](#appendix) of this specification provides an example usage of a 
+PayID Easy Checkout URL using Web Redirects, supplemental RFCs are needed to define the different ways in which a PayID
+client can utilize a PayID Easy Checkout URL.
+
+# PayID Easy Checkout Protocol
+The PayID Easy Checkout Protocol can be used to facilitate an end-to-end checkout flow between a recipient client, such
+as an online merchant UI, and a sending client, such as a wallet.
+
+The protocol is comprised of two parts:
+
+1. PayID Easy Checkout Discovery
+2. PayID Easy Checkout URL Assembly
+
+## PayID Easy Checkout Discovery
+PayID Easy Checkout extends [PAYID-DISCOVERY][] by defining a new link in the PayID metadata JRD
+returned by a PayID Discovery query. This link, defined in the [JRD section](#payid-easy-checkout-jrds) 
+of this specification, includes the PayID Easy Checkout URL representing a resource on the wallet which can
+be used to complete a payment.
+
+E-commerce merchants who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery server to 
+obtain a PayID Easy Checkout URL. Digital wallets and PayID server operators who wish to enable PayID Easy
```

Comment:
```suggestion
obtain a PayID Easy Checkout URL. Therefore, senders who wish to enable PayID Easy
```

Reply:
I guess this depends on what we define a "sender" to be.  I wanted to make a distinction between "sender" and "PayID Operator".  A "sender" alone cannot enable EZ Checkout, but a PayID server operator can.  Plus, this spec is really for PayID Servers, not the sender